src/gov/nasa/worldwind/geom/coords/MGRSCoordConverter.java

```diff
@@ -1103,7 +1103,7 @@ private UPSCoord convertMGRSToUPS(String MGRS)
             {
                 hemisphere = AVKey.SOUTH;
 
-                ltr2_low_value = upsConstants[mgrs.latitudeBand][12]; //.ltr2_low_value;
+                ltr2_low_value = upsConstants[mgrs.latitudeBand][1]; //.ltr2_low_value;
```

Comment:
Clearly a typo. See the pattern above, too. This is causing `IndexOutOfBoundsException`. upsConstants is a 2-dimensional array whose second dimension all have 6 elements.